components/webhook-mgt/org.wso2.carbon.identity.webhook.metadata/src/main/java/org/wso2/carbon/identity/webhook/metadata/api/constant/ErrorMessage.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.webhook.metadata.api.constant;
+
+/**
+ * Error messages for webhook metadata.
+ */
+public enum ErrorMessage {
+
+    // Client errors
+    PROFILE_NOT_FOUND("CLIENT-00002", "Profile not found",
```

Comment:
We haven't been following a pattern like CLIENT- or SERVER-

Rather we prefix for the component and have a range to separate client and server errors right.
Why we moved with a different format ?

Reply:
fixed with https://github.com/wso2/carbon-identity-framework/pull/6782/commits/d7e56f119890fd106b7e9d571d4dd8e08b41b0b0